tests/ami/windows/install_openssh.ps1

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script downloads and installs OpenSSH server from https://github.com/PowerShell/Win32-OpenSSH.OpenSSH.
+# Script requires following environment variables being set.
+#   PUBLIC_KEY - string that contains public key for future authorization.
+#
+
+
+
+function Expand-ZIPFile($file, $destination) {
```

Comment:
Is there some kind of PowerShell linter out there? Something similar to shellcheck which we use for bash / sh scripts.

Reply:
Looks like there is https://github.com/PowerShell/PSScriptAnalyzer, but no idea how useful it actually is.

Reply:
I don't think I so familiar with that too. But I'll have a look.